packages/next/build/webpack-config.ts

```diff
@@ -298,7 +299,7 @@ export default function getBaseWebpackConfig (dir: string, {dev = false, isServe
   }
 
   if (typeof config.webpack === 'function') {
-    webpackConfig = config.webpack(webpackConfig, { dir, dev, isServer, buildId, config, defaultLoaders, totalPages })
+    webpackConfig = config.webpack(webpackConfig, { dir, dev, isServer, buildId, config, defaultLoaders, totalPages, webpack })
```

Comment:
I think this is a great idea, but I'm not sure how this fits into the current PR 🤔

Reply:
I'm trying to fix #6353 and this is related to that